activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/process-extensions-schema.json

```diff
@@ -281,6 +282,37 @@
             }
           }
         },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "bigdecimal"
+              }
+            }
+          },
+          "then": {
```

Comment:
how did you validate the type and the pattern in the anyOf?

Reply:
I can add a validation pattern in the first element of the anyOf but I don't know if it's needed (please look at the "integer" case)